lib/PrintAsClang/_SwiftStdlibCxxOverlay.h

```diff
@@ -179,4 +179,194 @@ class Error {
   void *_Nonnull opaqueValue = nullptr;
 };
 
+namespace _impl {
+
+constexpr static std::size_t max(std::size_t a, std::size_t b) {
```

Comment:
can you make this `inline` instead of `static`, that's the more appropriate linkage specifier.